core/block.js

```diff
@@ -1673,7 +1676,14 @@ Blockly.Block.prototype.interpolate_ = function(message, args, lastDummyAlign) {
           input.setCheck(element['check']);
         }
         if (element['align']) {
-          input.setAlign(alignmentLookup[element['align']]);
+          var align = !!element['align'] && element['align'].toUpperCase();
+          var alignment = alignmentLookup[align];
+          if (alignment != null) {
```

Comment:
This will be undefined if it doesn't exist in the lookup, not null. Should be if (alignment)

Reply:
undefined == null, so this works. However if your suggestion is ``alignment != undefined`` is more appropriate here, then I think you're right. I initially avoided it because I know undefined can be re-assigned in old browsers, so the more appropriate check here would be ``typeof alignment !== "undefined"``, but that's a few more characters than ``undefined == null``. Looking into this further, it turns out ES5 is when the issue of re-assignment of undefined was fixed, and since we support only ES5+ browsers, then going with ``alignment != undefined`` sounds good.